projects/rocfft/library/src/rtc_stockham_kernel.cpp

```diff
@@ -29,6 +29,27 @@
 
 #include "device/kernel-generator-embed.h"
 
+static StockhamPartialPassParams get_partial_pass_params(const TreeNode&  node,
+                                                         const FFTKernel& kernel)
+{
+    if(node.isPartialPassEnabled())
+    {
+        StockhamPartialPassParams pp_params;
+
+        pp_params.off_dim         = node.ppOffDim;
+        pp_params.current_dim     = node.ppCurrDim;
+        pp_params.pp_factors_curr = std::vector<unsigned int>(
+            kernel.pp_params.pp_factors_curr.begin(), kernel.pp_params.pp_factors_curr.end());
+        pp_params.pp_factors_other = std::vector<unsigned int>(
+            kernel.pp_params.pp_factors_other.begin(), kernel.pp_params.pp_factors_other.end());
+        pp_params.parent_length = std::vector<unsigned int>(node.length.begin(), node.length.end());
```

Comment:
```suggestion
        pp_params.pp_factors_curr = kernel.pp_params.pp_factors_curr;
        pp_params.pp_factors_other = kernel.pp_params.pp_factors_other;
        pp_params.parent_length = node.length;
```
any reason that can't be done instead?

Reply:
The element type of these vectors are not the same.

Reply:
```suggestion
        pp_params.pp_factors_curr.assign(
            kernel.pp_params.pp_factors_curr.begin(), kernel.pp_params.pp_factors_curr.end());
        pp_params.pp_factors_other.assign(
            kernel.pp_params.pp_factors_other.begin(), kernel.pp_params.pp_factors_other.end());
        pp_params.parent_length.assign(node.length.begin(), node.length.end());
```
avoids the construction of a temporary RHS then.

Reply:
Sounds good, new commit should have your suggestion.